### ISSUE TITLE
As a server, I want the time to live (TTL) on the server response headers to be longer

### ISSUE THREAD

**Issue by hollyallen** [18F/college-choice#1341]
Please increase the TTL on the server responses to 1 hour from 5 minutes.

**Comment by LisaGee** [comment]
@hollyallen Can you confirm if we did this? I feel like we might have, no?

**Comment by hollyallen** [comment]
Yes and no. This pull request increased the TTL, but we haven't done another production deployment to get that change out. https://github.com/18F/open-data-maker/pull/212

**Comment by abisker** [comment]
@yozlet should we include in the production deployment next week to close out?

**Comment by ultrasaurus** [comment]
via @yozlet: there are cases when we want to reduce the CACHE_TTL (for example, in advance of a data update - otherwise it's going to make it harder to check that the new data's being picked up). Perhaps CACHE_TTL should be picked up from an environment variable?

I like this idea.  Will close the PR, and recommend this approach when this becomes a priority (probably if there's a big announce around a future data update)
